i18n/si.json

```
@@ -0,0 +1,297 @@
+{
+
+"@metadata": {
+
+"කර්තෘ": [
```

This cannot be translated.
```suggestion
"authors": [
```